tests/UI/campaigns/functional/BO/03_catalog/04_attributesAndFeatures/01_attributes/06_changeAttributePosition.js

```diff
@@ -0,0 +1,181 @@
+require('module-alias/register');
+
+// Helpers to open and close browser
+const helper = require('@utils/helpers');
+
+// Common tests login BO
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const attributesPage = require('@pages/BO/catalog/attributes');
+const viewAttributePage = require('@pages/BO/catalog/attributes/view');
+
+// Import data
+const {Attributes} = require('@data/demo/attributes');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shopParameters_catalog_attributesAndFeatures_attributes_changeAttributePosition';
```

Comment:
```suggestion
const baseContext = 'functional_BO_catalog_attributesAndFeatures_attributes_changeAttributePosition';
```

Reply:
Thanks for noticing this, there was the same error on all tests in this directory.

Reply:
done, thanks